DuckDuckGo/AppIconSettingsCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  IconSettingsCell.swift
```

Comment:
Wrong file name :)

Reply:
I really tried not to miss any :D Thanks